src-ui/components/MultiScreen.cpp

```diff
@@ -181,6 +181,11 @@ void MultiScreen::layout()
 
 void MultiScreen::onVoiceInfoChanged() { sample->repaint(); }
 
+void MultiScreen::updateSamplePlaybackPosition(int64_t samplePos)
+{
+    sample->updateSamplePlaybackPosition(samplePos);
+}
```

Comment:
Also do we want some sort of sample is visible check somewhere in this path? Or do we have one already?

Reply:
Good point. I have added a check in MappingPane::updateSamplePlaybackPosition